SourceCode/draw-circle/main.cpp

```diff
@@ -0,0 +1,49 @@
+#include <iostream>
+#include <fstream>
+
+#include "view.h"
+#include "cuda_array.cuh"
+#include "cuda_map.cuh"
+
+map_data read_data(){
+        // TODO read JSON
+        // 
+        // tower: circle
+        // poi: rectangle, which will intersect the circle. Think of this a that part of the canvas we are allowed to draw on
```

Comment:
Note to reviewer: POI is from the task on the admission test. Feel free to ignore it for the purpose of drawing.